README.md

```diff
@@ -2,12 +2,24 @@ Keep and restore fcitx state for each buffer separately when leaving/re-entering
 
 D-Bus only works with the same user so this won't work with `sudo vim`. See the `fcitx5-server` branch for an experimental implementation that supports `sudo vim`.
 
-Requires:
+By default, it use python3 and D-Bus to toggle IME state.
+If you set `g:fcitx5_remote` to the executable path of `fcitx5-remote` **BEFORE** loading the plugin, it will use `fcitx5-remote` instead of python and D-Bus. In this case, python3 support is optional.
+
+Usually `fcitx5-remote` mode is way faster since python script need 400ms+ for the initial load.
```

Comment:
```suggestion
Usually `fcitx5-remote` mode is way faster since Python script need quite some time for the initial load if you don't use any other plugins that load Python.
```

The loading time is largely affected by disk speed, I/O cache and installed Python libraries.

Reply:
> ... if you don't use any other plugins that load Python.

Does other plugins (assume to be vim plugins) matter?
I tried `python3 -c 'import dbus'` and it already takes about 170ms (when files are already cached), comparing to `python3 -c ''`'s 80ms. I think the cost is mostly on `import`s rather than python3 startup time.

Reply:
42ms vs 30ms here.

Python's import machinery is complicated and your distribution may have code hooking inside it. You may also check if cached pyc is correctly used.

Vim only needs to initialize Python once, so if you have multiple Python plugins, you already pay the time.